imports/api/Players.js

```diff
@@ -3,4 +3,4 @@ import { Mongo } from 'meteor/mongo';
 //Until some actually halfway programming happens players 
 //must be created and edited through the actual database
 // >>This might not be a bad thing
-export const Players = new Mongo.Collection('players');
\ No newline at end of file
+export default new Mongo.Collection('players');
```

Comment:
Actually, it's interesting.

When you have some code like this:
```javascript
export const Players = new Mongo.Collections('players');
```

You have to explicitly name the thing you export by using `{ }`. However, by declaring something as the default export for a file, such as:
```javascript
export default new Mongo.Collections('players');
```

You do not explicitly name the export. To get players, you can name the variable absolutely anything, and it will work.

Examples:
```javascript
import Players from './Players';
Players.find({});
```

```javascript
import blah from './Players';
blah.find({});
```
These would both work.